clock/app/views/stores/_form.html.erb

```diff
@@ -33,6 +33,8 @@
       </div>
     </div>
 
+    <%= form.hidden_field :status, value: true %>
```

Comment:
check the indentation in this part, also you can use the bootstrap_form gem
[https://github.com/bootstrap-ruby/bootstrap_form](https://github.com/bootstrap-ruby/bootstrap_form)